tests/secrets/https.test.ts

```diff
@@ -60,4 +61,38 @@ describe("Secrets HTTPS (OAuth)", () => {
     await secretClient.setSecret(secretName, secretValue);
     await secretClient.beginDeleteSecret(secretName);
   });
+  it(`Should work using HTTPS endpoint and return correct values @loki`, async () => {
+    const token = generateJWTToken(
```

Comment:
This would be better placed in `api/secrets.test.ts` file.

`https.test.ts` is for HTTPS specific tests.